tutorials/cifar10-distributed.ipynb

```diff
@@ -0,0 +1,1534 @@
+    "2. Else save the models to disk using [`DiskSaver()`](https://pytorch.org/ignite/generated/ignite.handlers.DiskSaver.html#ignite.handlers.DiskSaver)."
+   ],
+   "metadata": {
+    "id": "jp6sWINP9CAq"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "source": [
+    "def get_save_handler(config):\n",
+    "    if config[\"with_clearml\"]:\n",
+    "        from ignite.contrib.handlers.clearml_logger import ClearMLSaver\n",
+    "\n",
+    "        return ClearMLSaver(dirname=config[\"output_path\"])\n",
+    "\n",
+    "    return DiskSaver(config[\"output_path\"], require_empty=False)"
+   ],
+   "outputs": [],
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-14T17:01:20.256676Z",
+     "iopub.status.busy": "2021-09-14T17:01:20.256397Z",
+     "iopub.status.idle": "2021-09-14T17:01:20.262069Z",
+     "shell.execute_reply": "2021-09-14T17:01:20.261000Z",
+     "shell.execute_reply.started": "2021-09-14T17:01:20.256647Z"
+    },
+    "id": "-DG0Pj4pJJFw"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Resume from Checkpoint\n",
+    "\n",
+    "If a checkpoint file path is provided, we can resume training from there by loading the file."
+   ],
+   "metadata": {
+    "id": "T1N1iR2f9R0n"
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "source": [
+    "def load_checkpoint(resume_from):\n",
+    "    checkpoint_fp = Path(resume_from)\n",
+    "    assert (\n",
+    "        checkpoint_fp.exists()\n",
+    "    ), f\"Checkpoint '{checkpoint_fp.as_posix()}' is not found\"\n",
+    "    logger.info(f\"Resume from a checkpoint: {checkpoint_fp.as_posix()}\")\n",
+    "    checkpoint = torch.load(checkpoint_fp.as_posix(), map_location=\"cpu\")\n",
+    "    return checkpoint"
+   ],
+   "outputs": [],
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-09-14T17:01:20.613959Z",
+     "iopub.status.busy": "2021-09-14T17:01:20.613673Z",
+     "iopub.status.idle": "2021-09-14T17:01:20.620012Z",
+     "shell.execute_reply": "2021-09-14T17:01:20.618881Z",
+     "shell.execute_reply.started": "2021-09-14T17:01:20.613923Z"
+    },
+    "id": "j9La55z97PVa"
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Create Trainer\n",
+    "\n",
+    "Finally, we can create our `trainer` in four steps:\n",
+    "1. Choose whether to enable [Automatic Mixed Precision](https://pytorch.org/docs/stable/amp.html) (AMP) or not. Enabling AMP will speed up computations on large neural networks and reduce memory usage while retaining performance. A [`GradScaler()`](https://pytorch.org/docs/stable/amp.html#torch.cuda.amp.GradScaler) object also has be created internally to scale the `loss` so that gradients don't round-up to zero while training (`scaler=True`). \n",
```

Comment:
I would exchange point 1 and 2 as creating the trainer is the most important and AMP is optional and secondary. I'd also remove "A [`GradScaler()`](https://pytorch.org/docs/stable/amp.html#torch.cuda.amp.GradScaler) object also has be created internally to scale the `loss` so that gradients don't round-up to zero while training (`scaler=True`)." as IMO it a detail.